EZP-30751: DFS file permissions are ignored (#941)
ezsystems/developer-documentation: docs/guide/file_management.md

    Make sure to configure permissions using a number and *not* a string. If you write "0644" it will *not* be interpreted by PHP as an octal number, and unexpected permissions will be applied.

!!! note

>     In case when NFS [adapter](file_management.md#adapter) is in use, you should configure file permissions under `oneup_flysystem` key instead, as follows:

Reviewer: ```suggestion
    When using the NFS [adapter](file_management.md#adapter), configure file permissions under the `oneup_flysystem` key instead, as follows:
```
